lib/taiko.js

```diff
@@ -70,6 +70,10 @@ module.exports.openBrowser = async (options = { headless: true }) => {
 /**
  * Allows to switch between tabs using URL or page title.
  * 
+ * @example
+ * switchTo('https://taiko.gauge.org/') - switch using URL
+ * switchTo('Taiko') - switch using Title
```

Comment:
@NivedhaSenthil what if we don't know the URL to switch as it might be dynamic URL?

Reply:
We can switch using Title in that case.

Reply:
what if two tabs with the same title are present ? how would taiko handle that

Reply:
Taiko will pick up the first matching one.

Reply:
Is there a way to select the specific window handle currently?

Reply:
There is nothing like window handle in taiko for now.